app/controllers/ems_cluster_controller.rb

```diff
@@ -122,12 +122,17 @@ def set_config(db_record)
   end
 
   def breadcrumbs_options
+    breadcrumbs = [
+      {:title => _("Compute")},
+      {:title => _("Infrastructure")},
+      {:title => _("Clusters / Deployment Roles")}
+    ]
+    header_title = title_for_clusters
+    breadcrumbs.push(:title => header_title) unless header_title == "Clusters / Deployment Roles"
```

Comment:
`header_title == "Clusters / Deployment Roles"` won't work, since result of `header_title()` is always localized.